x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_table.test.tsx

```diff
@@ -140,4 +172,75 @@ describe('EnginesTable', () => {
       expect(tableContent).not.toContain('Universal');
     });
   });
+
+  describe('actions', () => {
+    it('when AppLogic.values.myRole.canManageEngines is false actions are hidden', () => {
+      jest.clearAllMocks();
+      setMockValues({
+        // AppLogic
```

Comment:
We don't do this elsewhere, I don't think you necessarily need to use this comment here.

Reply:
I find this really useful personally, since we're unable to setMockValues with individual logics (luckily we've never had a logic value or action collision).

Reply:
That's a good point